data/mods/Russ_Extra_Stuff/items/container-Buckets.json

```diff
@@ -0,0 +1,82 @@
+[
+ {
+        "id": "bucket_storage",
+        "type": "CONTAINER",
+        "symbol": ")",
+        "color": "dark_gray",
+        "name": "5 gallon bucket",
+        "category": "other",
+        "description": "A five gallon bucket with a lid like the kind you used to fill with mackerel when you were fishing with your father.  Could be used for liquids or storage, just activate it to switch between modes.  Right now you're using it for wearable storage.",
+        "price": 1000,
+        "weight": 155,
+        "volume": 7,
+        "bashing": 1,
+        "cutting": 0,
+        "to_hit": -1,
+        "material": "plastic",
+        "contains": 75,
+        "armor_data" : {
+        "covers" : ["HANDS"],
+        "coverage" : 5,
+        "material_thickness" : 4,
+        "encumbrance" : 30,
+        "environmental_protection" : 0,
+        "warmth" : 0,
+        "storage" : 25
+        			  },
+        "rigid": true,
+        "watertight": true,
+        "seals": true,
+        "flags": ["STURDY"],
+
+    "use_action": {
+        "type": "auto_transform",
+        "msg": "You decide to use this bucket as container for liquids.",
+        "target": "bucket_liquids",
+        "need_charges": 0
+    			}
+},
+{    "id": "bucket_liquids",
+        "type": "CONTAINER",
+        "symbol": ")",
+        "color": "dark_gray",
+        "name": "5 gallon bucket (liquids)",
+        "category": "other",
+        "description": "A five gallon bucket like the kind you used to fill with mackerel when you were fishing with your father.  You're using this one for liquids, activate it to use it as wearable item storage instead.",
+        "price": 1000,
+        "weight": 155,
+        "volume": 20,
+        "bashing": 1,
+        "cutting": 0,
+        "to_hit": -1,
+        "material": "plastic",
+        "contains": 75,
```

Comment:
Thank you i am making all the changes.
On Aug 4, 2015 10:05 AM, "Coolthulhu" <notifications@github.com> wrote:

> In data/mods/Russ_Extra_Stuff/items/container-Buckets.json
> <https://github.com/CleverRaven/Cataclysm-DDA/pull/13075#discussion_r36190985>
> :
>
> > +},
> > +{    "id": "bucket_liquids",
> > +        "type": "CONTAINER",
> > +        "symbol": ")",
> > +        "color": "dark_gray",
> > +        "name": "5 gallon bucket (liquids)",
> > +        "category": "other",
> > +        "description": "A five gallon bucket like the kind you used to fill with mackerel when you were fishing with your father.  You're using this one for liquids, activate it to use it as wearable item storage instead.",
> > +        "price": 1000,
> > +        "weight": 155,
> > +        "volume": 20,
> > +        "bashing": 1,
> > +        "cutting": 0,
> > +        "to_hit": -1,
> > +        "material": "plastic",
> > +        "contains": 75,
>
> It still has "contains" in liquid mode.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CleverRaven/Cataclysm-DDA/pull/13075/files#r36190985>.
>